core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -130,6 +169,37 @@ export class DotBlockEditorComponent implements OnInit, OnDestroy {
         this.editor.view.dispatch(tr);
     }
 
+    /**
+     * This methods get the customBlocks variable to retrieve the custom modules as Objects.
+     * Validates that there is customBlocks defined.
+     */
+
+    private async getCustomBlocks(): Promise<AnyExtension[]> {
+        let data = {};
```

Comment:
I think we can do this. This way, we can avoid using `data as CustomBlock` in different places.

```suggestion
        let data: CustomBlock;
```